script/Arcueid/Day 8/08_00_ARC08_1.txt

```diff
@@ -278,7 +278,7 @@ This is the price I pay for getting involved with vampires as a human.
 [sha:a75fa4f7d32d4f1a8155dbfbe98eb535b025a6ed]{
 -- Page 15, Offset 19715.
 -- 吸血鬼と対峙する時に危険を冒すのではなく、常から危険を冒していく。
-The risks I'm taking aren't confined to the times I'm actually going toe to toe with a vampire,
+The risks I'm taking aren't confined to the times I'll actually going toe to toe with a vampire,
```

Comment:
I _think_ original reads better to me but have confused myself at this point. If we do change it you need a `be` somewhere though.

Reply:
I prefer original